src/Persistence/PaYaml/Models/SchemaV3/ThemeDefinition.cs

```diff
@@ -5,9 +5,30 @@ namespace Microsoft.PowerPlatform.PowerApps.Persistence.PaYaml.Models.SchemaV3;
 
 public record ThemeDefinition
 {
-    public required string BasePaletteColor { get; init; }
+    public required string? BasePaletteColor { get; init; }
     public int? HueTorsion { get; init; }
     public int? Vibrancy { get; init; }
-    public bool ThemePrimaryColorLocked { get; init; }
-    public string[]? ColorOverrides { get; init; }
+    public bool? ThemePrimaryColorLocked { get; init; }
+    public ColorOverrides? ColorOverrides { get; init; }
+    public string? Font { get; init; }
```

Comment:
I usually make complex properties occur after simple properties. It just makes the yaml easier to read.
Since the Font is pretty important, I'd consider making it the 1st property. Since all other properties deal with color manipulation.